.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
We have a .prettierrc.json in edge-core-js that we should copy over here instead. We definintely don't want the `"printWidth": 120` line.